editor/gui/editor_bottom_panel.cpp

```diff
@@ -100,6 +100,15 @@ void EditorBottomPanel::_update_disabled_buttons() {
 	right_button->set_disabled(h_scroll->get_value() + h_scroll->get_page() == h_scroll->get_max());
 }
 
+void EditorBottomPanel::_ensure_control_visible(Control *p_control) {
+	if (!p_control || !p_control->is_visible()) {
```

Comment:
If there are no cases of a button being hidden in the bottom panel then we can remove this check `!p_control->is_visible()`.